service-runner/src/main/resources/config/assets.yaml

```diff
@@ -19,11 +19,14 @@ assets:
       methods:
         - bank_account
         - cash
-    send:
+    send: # (Deprecated) All fields should be incorporated into `sep31`.
       fee_fixed: 0
       fee_percent: 0
       max_amount: 1000000
     sep31:
```

Comment:
Is it better to merge `sep31` into `send` instead? That way, we have `deposit/withdraw/send` instead of `deposit/withdraw/sep31`.

Reply:
I think `sep31` is a more intuitive choice since I have been working on SEP-12 and SEP-31 doc for while ( :stuck_out_tongue: ) and `sep31` is a more commonly referenced term in out doc.

Reply:
Hmm... I feel like we should be updating the docs to use `send` instead. What are your thoughts on this @Ifropc?

Reply:
Hm I agree that SEP-31 is for me personally more clear than `send`, but I like the consistency of deposit/withdraw. Maybe we should support both?

Reply:
I think `sep31` is a bit more clear, including the code. I'd also add `sep24` in withdrawal/deposit too, so it's `sep24.deposit` `sep24.withdrawal` for consistency. WDYT?

Reply:
Isn't deposit and withdraw currently shared by sep6 and sep24? Do we need to separate them?

Reply:
```
sep31:
   ...
sep24:
   withdraw:
      ...
    deposit:
      ...
sep6:
   withdraw:
      ...
    deposit:
      ...
```